cnames_active.js

```diff
@@ -1639,6 +1639,7 @@ var cnames_active = {
   "live-editor": "geard-dev.github.io/web-based-live-code-editor",
   "liveflow": "hasharray.github.io/liveflow.js",
   "livenotif": "siffreinsg.github.io/livenotif",
+  "livevalidate":"thedhanawada.github.io/LiveValidateJS"
```

Comment:
As the CI error notes, you are missing a space between the key/value, and a comma at the end of the line
```suggestion
  "livevalidate": "thedhanawada.github.io/LiveValidateJS",
```